ui/browser/actions/main-actions.js

```diff
@@ -46,3 +46,7 @@ export function setPageOrder(pageOrder) {
 export function setPageAreaVisibility(visible) {
   return { type: types.SET_PAGE_AREA_VISIBILITY, visible };
 }
+
+export function setWebviewScroll(y) {
```

Comment:
Adding horizontal scroll would be easy, but yeah atleast make the API capable when/if we need by taking in a `{ y }` arg instead of directly